ex2_system/source/system_tasks.c

```diff
@@ -34,14 +34,18 @@
  *  error report of daemon creation
  */
 SAT_returnState start_system_tasks(void) {
-    TaskHandle_t _ = 0;
-    if (start_beacon_daemon() != SATR_OK || start_coordinate_management_daemon() != SATR_OK ||
-                start_diagnostic_daemon() != SATR_OK || start_housekeeping_daemon() != SATR_OK ||
-                start_system_stats_daemon() != SATR_OK || start_state_daemon() != SATR_OK ||
-        start_task_manager() != SATR_OK || start_logger_daemon(_) != SATR_OK) {
-        ex2_log("Error starting system tasks\r\n");
-        return SATR_ERROR;
-    }
-    ex2_log("All system tasks started\r\n");
-    return SATR_OK;
+
+  TaskHandle_t _;
+  if (//start_beacon_daemon() != SATR_OK ||

```

Comment:
We should get one of the new software members to improve this

Reply:
Yes that would be good so that we can stop commenting/uncommenting